### PR TITLE
Adding ptflops

### PR DESCRIPTION
**Pull request by cnpcshangbo**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by cnpcshangbo** [comment]
@conda-forge/help-python ready for review.

File: recipes/ptflops/conda_build_config.yaml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+channels:
```
Comment (synapticarbors) [inline recipes/ptflops/conda_build_config.yaml:1]: conda-forge packages should not depend on other non-conda-forge channels. Please remove this. I believe all of the dependencies are available on the `conda-forge` channel.

File: recipes/ptflops/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+build:
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+    - torchvision
```
Comment (synapticarbors) [inline recipes/ptflops/meta.yaml:21]: `torchvision` doesn't looks like it's required to build the package, so it should be removed from the `host` section

File: recipes/ptflops/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  run:
+    - python >=3.6
+    - pytorch
+    - torchvision
```
Comment (synapticarbors) [inline recipes/ptflops/meta.yaml:25]: `torchvision` appears to be an optional dependency listed under the optional `dev` section. Unless there is a compelling reason, I'd remove it here.

File: recipes/ptflops/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+test:
```
Comment (synapticarbors) [inline recipes/ptflops/meta.yaml:27]: Please add a `pip check` test. See:
https://github.com/conda-forge/staged-recipes/blob/main/recipes/example/meta.yaml#L61-L64

**Review by synapticarbors — CHANGES_REQUESTED** [review]

**Comment by cnpcshangbo** [comment]
@conda-forge/help-python, ready for review!

File: recipes/ptflops/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,40 @@
+{% set name = "ptflops" %}
+{% set version = "0.7.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://files.pythonhosted.org/packages/95/3d/d2af80d6b90d74f1b73e688c287434912fc4416517c6f7505bb6d2ca77d1/ptflops-0.7.3.tar.gz
```
Comment (synapticarbors) [inline recipes/ptflops/meta.yaml:9]: Sorry I missed this in my original review, but could you use our canonical `url` definition:
https://github.com/conda-forge/staged-recipes/blob/main/recipes/example/meta.yaml#L16

With it hardcoded this way, I'm not sure if the bots that automatically submit PRs when the package is updated upstream will be able to work.

**Comment by cnpcshangbo** [comment]
@conda-forge/help-python, ready for review!

**Pull request merged**